stale-projects.js

```diff
@@ -0,0 +1,9 @@
+exports.getStaleProjects = (currentProjects, targetProjects) => (
+  currentProjects.filter((p) => !targetProjects.includes(p.node.project.name))
+);
+
+exports.removeStaleProjects = async ({ client, projects }) => (
+  Promise.all(projects.map((staleProject) => (
+    client.rest.projects.deleteCard({ card_id: staleProject.node.databaseId })
+  )))
+);
```

Comment:
there is a risk of being throttled by the API when running all the requests in parallel. thus, a classic loop with `await` might be better option for this case, i.e. making this less efficient on purpose:

```suggestion
exports.removeStaleProjects = async ({ client, projects }) => {
  for (const project of projects) {
    await client.rest.projects.deleteCard({ card_id: project.node.databaseId });
  }
};
```